release_docs/RELEASE.txt

```diff
@@ -463,6 +463,23 @@ New Features
 
     Library:
     --------
+    - Change how release part of version is checked for compatibility
```

Comment:
I think release has two meanings here which might be misleading.. How about:

- Change how release part of major.minor.release version is checked for compatibility

should be "is compatible in line 2 of the first paragraph (sorry, my fault).

Second paragraph  - library release doesn't have to be newer, just not older, so maybe "first checks that the library version matches major and minor in the header version, and the library release number is not less than the release number in the headers."